test/performance/odp_bench_misc.c

```diff
@@ -698,7 +719,8 @@ bench_info_t test_suite[] = {
 	BENCH_INFO(time_diff, init_time_local, 0, "time_diff (local)"),
 	BENCH_INFO(time_diff_ns, init_time_global, 0, NULL),
 	BENCH_INFO(time_sum, init_time_global, 0, NULL),
-	BENCH_INFO(time_to_ns, init_time_global, 0, NULL),
+	BENCH_INFO(time_to_ns_short, init_time_global, 0, "time_to_ns (short)"),
+	BENCH_INFO(time_to_ns_long, init_time_global, 0, "time_to_ns (long)"),
```

Comment:
Since init function is not needed anymore in v2, the init function pointer should be set to NULL. With that change this looks ok, as long as PR 1959 goes in first (because init is no longer touching the source data, the improved warmup is needed to bring it to cache before the actual test run).

Reply:
Init set to NULL in v3